### DISCUSSION TITLE
ViewerEnvironment not available...

### DISCUSSION THREAD

**Discussion by ghost** [leap71/PicoGK#56] [Q&A]
![ScreenshotQ](https://github.com/user-attachments/assets/2a28a2ea-1464-492f-889e-cfb09893f4aa)

It basically says, the zip-Folder "ViewerEnvironment.zip" doesn't exist, which is true, up until "net8.0" everything looks right... I Had no issues with installation everything went clean. How can I then manually add the environment.zip folder ?

Thanks in advance.

**Comment by LinKayser** [comment]
Here's some pointers: https://github.com/leap71/PicoGK/blob/main/Documentation/README.md#my-viewer-is-dark

Basically, your project structure needs to have the PicoGK folder as subdirectory of the main source code directory. If you want a different structure, you can supply the path to the viewer.

Best,
Lin

**Reply by ghost** [reply]
Hey Lin,
thanks for the fast reply! But it seems to me that my project structure is exactly like required. I have the PicoGK Folder as a subdirectory to my project folder named PicoGK Example. Or am I misunderstanding something?
In the picture I sent with the message from last week, it says in the terminal: "Could not find file 'C:\Users\peter\PicoGK Example\bin\Debug\net8.0\ViewerEnvironment.zip'.", I thought it might have something to do with that, because I indeed do not have any Zip-file in the net8.0 folder.

Best,
Peter
![WhatsApp Bild 2024-11-18 um 21 53 23_9002f7b4](https://github.com/user-attachments/assets/b210d815-fc5f-4dda-9fb5-1f68386c6d1e)
